packages/frontend/src/List.svelte

```diff
@@ -0,0 +1,152 @@
+import BootcCurrentProgress from './lib/BootcCurrentProgress.svelte';
+import { Unsubscriber } from 'svelte/store';
+import { searchPattern, filtered } from './stores/historyInfo';
+import BootcIcon from './lib/BootcIcon.svelte';
+import FilteredEmptyScreen from './lib/upstream/FilteredEmptyScreen.svelte';
+import BootcEmptyScreen from './lib/BootcEmptyScreen.svelte';
+import BootcFolderColumn from './lib/BootcFolderColumn.svelte';
+
+// Search functionality
+export let searchTerm = '';
+$: searchPattern.set(searchTerm);
+
+let history: BootcBuildInfoWithSelected[] = [];
+let historyUnsubscribe: Unsubscriber;
+
+interface BootcBuildInfoWithSelected extends BootcBuildInfo {
+  selected: boolean;
+}
+
+onMount(async () => {
+  historyUnsubscribe = filtered.subscribe(value => {
+    history = value.map(build => ({ ...build, selected: false }));
+  });
+});
+
+onDestroy(() => {
+  historyUnsubscribe();
+});
+
+// Bulk delete the selected builds
+let bulkDeleteInProgress = false;
+async function deleteSelectedBuilds() {
+  const selected = history.filter(history => history.selected);
+  if (selected.length === 0) {
+    return;
+  }
+
+  // mark builds for deletion
+  bulkDeleteInProgress = true;
+
+  // Delete all the selected builds
+  await bootcClient.deleteBuilds(selected);
+  bulkDeleteInProgress = false;
+}
+
+async function gotoBuild(): Promise<void> {
+  router.goto('/build');
+}
+
+let selectedItemsNumber: number;
+let table: Table;
+
+// COLUMNS
+let statusColumn = new Column<BootcBuildInfo>('Status', {
+  align: 'center',
+  width: '70px',
+  renderer: BootcStatus,
+});
+
+let progressColumn = new Column<BootcBuildInfo>('Progress', {
+  width: '2fr',
+  overflow: true,
+  renderer: BootcCurrentProgress,
+});
+
+let imageColumn = new Column<BootcBuildInfo>('Image', {
+  width: '2fr',
+  renderer: BootcNameColumn,
+  comparator: (a, b) => a.name.localeCompare(b.name),
+});
+
+let typeColumn = new Column<BootcBuildInfo, string>('Type', {
+  renderMapping: object => object.type,
+  renderer: SimpleColumn,
+  comparator: (a, b) => a.type.localeCompare(b.type),
+});
+
+let archColumn = new Column<BootcBuildInfo, string>('Arch', {
+  renderMapping: object => object.arch,
+  renderer: SimpleColumn,
+  comparator: (a, b) => a.arch.localeCompare(b.arch),
+});
+
+let folderColumn = new Column<BootcBuildInfo>('Folder', {
+  renderer: BootcFolderColumn,
+  comparator: (a, b) => a.folder.localeCompare(b.folder),
+});
+
+const columns: Column<BootcBuildInfo, BootcBuildInfo | string>[] = [
+  statusColumn,
+  progressColumn,
+  imageColumn,
+  typeColumn,
+  archColumn,
+  folderColumn,
+  new Column<BootcBuildInfo>('Actions', { align: 'right', renderer: BootcColumnActions, overflow: true }),
+];
+
+const row = new Row<BootcBuildInfo>({
+  selectable: _build => true,
+});
+</script>
+
+<NavPage bind:searchTerm="{searchTerm}" title="BootC (Bootable Container) Image Builder" searchEnabled="{true}">
```

Comment:
The title is long, and maybe even a bit limiting to just the image builder. Maybe just 'Bootable Containers' like the extension?